packages/dds/sequence/src/test/createSnapshotFiles.ts

```diff
@@ -8,7 +8,9 @@ import { convertSummaryTreeToITree } from "@fluidframework/runtime-utils";
 import { generateStrings, LocationBase } from "./generateSharedStrings";
 
 for (const s of generateStrings()) {
-    const summaryTree = s[1].getAttachSummary().summary;
+    const summaryTree = s.expected.getAttachSummary().summary;
```

Comment:
nit: I generally like destructuring arguments immediately when their aggregation doesn't have a great name and they're somewhat one-off. You can do that with `for const { expected, snapshotIsNormalized, snapshotPath } of generateString()) {`